### ISSUE TITLE
Option to read text boxes via text to speech

### ISSUE THREAD

**Issue by Bugsbane** [Secretchronicles/TSC#526]
It would be fantastic to have an option to have text boxes be read aloud via some text-to-speech algorithm, for both players too young to read them easily (say 3-8 years old) and also people whose eyesight is good enough to play, but not to read fine details.

Currently, every time my 4 year old gets to a text box, he needs to call me in to help him. He doesn't need my help anywhere else in the game (since he learned to turn Omega Mode on, at least).

On Mac / Windows, there are already good built in text to speech support. On Linux we could just use something like Flite as an optional dependency (hear the surprisingly decent online demo [here](http://tts.speech.cs.cmu.edu:8083/) and there's a ruby verion on GitHub [here](https://github.com/kubo/ruby-flite)).

**Comment by Quintus** [comment]
This is an interesting idea, but we need to consider how well this plays with #120. Not all languages have speech synthesizers of the same quality as English. I would (if GitHub allowed, heh) assign a higher priority to #120 than to this issue, because of the greater audience reached and maybe due to architectural changes #120 requires that will reflect on how to implement a speech synthesizer.

Then there has been the suggestion to have actual, real people speak the text and record it as sound files. This would probably conflict with a synthetic speech as suggested in this issue.

Valete,
Quintus

**Comment by Bugsbane** [comment]
Yes, I'd put #120 higher, too. While I acknowledge that we shouldn't put a lot of dev effort into anything that isn't usable by a large amount of our players, this seems like it could benefit a good number of players with little effort, without getting in the way of any of the other users. Nonetheless, I would understand this being low priority.

**Comment by datahead8888** [comment]
I think a text to speech tool might conflict with voice narration if we go that route -- speed synthesizers usually sound artificial.

**Comment by Bugsbane** [comment]
Well just have it where there isn't a recording available
